Classes/Service/FCEFootnoteService.php

```diff
@@ -70,6 +70,14 @@ public function renderItemList($content, $conf = array())
         return $this->getRenderingService()->renderFootnotes(explode(',', $footnoteUids));
     }
 
+    /**
+     * @param ContentObjectRenderer $cObj
+     */
+    public function injectContentObjectRenderer(ContentObjectRenderer $cObj)
```

Comment:
Should be constructor injection https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/DependencyInjection/Index.html